addon/components/new-bourbon-select-field.js

```diff
@@ -57,7 +57,7 @@ export default Component.extend(ClickHandlerMixin, {
   },
 
   clickHandler(e) {
-    if (!e.target.isEqualNode(document.activeElement)) {
+    if (!e.target.isEqualNode(document.activeElement) || document.activeElement.textContent !== this.get('label')) {
```

Comment:
can you elaborate on what this line does?

Reply:
this is the clickhandler to close the list when a user clicks outside of the element or opens another dropdown so that we only have 1 dropdown open at at time.

Reply:
i guess i was wondering more about why we're comparing text content on this line
```
document.activeElement.textContent !== this.get('label')
```